src/Microsoft.SqlTools.Hosting/Hosting/Protocol/RequestContext.cs

```diff
@@ -5,6 +5,7 @@
 
 using System.Threading.Tasks;
 using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using ErrorContract = Microsoft.SqlTools.Hosting.Contracts.Error;
```

Comment:
>ErrorContract [](start = 6, length = 13)

IMHO, I find this a little confusing since we're special case this one contract type...I prefer we leave it as Error.